corehq/apps/cloudcare/static/cloudcare/js/util.js

```diff
@@ -43,7 +43,7 @@ hqDefine('cloudcare/js/util', function () {
         if (message === undefined) {
             return;
         }
-        _show(message, $el, 5000, "alert alert-warning");
+        _show(message, $el, 30000, "alert alert-warning");
```

Comment:
30 seconds is a long enough delay that you might want to just not have this disappear at all (by passing `null`).

Reply:
Nick and I went back and forth on that, and I think he's pushing a bit for persistence. 

I'm pretty worried, though about the lack of associability between this notice and the individual form, and them either stacking, or someone filling out a form and not realizing this notice applied to the previous one.

If they're still having issues after this change, will review at a high level and see if there are ways to rethink